integration

```diff
@@ -856,6 +856,7 @@
   "krasnoukhov/homeassistant-smart-maic",
   "krasnoukhov/homeassistant-tesy",
   "krissen/polleninformation",
+  "krozgrov/ha-omlet-integration",
```

Comment:
This file is not valid in the current release.
https://github.com/krozgrov/ha-omlet-integration/blob/2025.6.1/hacs.json